<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>dashboardDialog</class>
 <widget class="QDialog" name="dashboardDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1171</width>
    <height>947</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>300</width>
    <height>750</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="dashwidget" native="true">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="styleSheet">
      <string notr="true">QWidget#dashwidget{
	background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0.323383 rgba(190, 159, 200, 255), stop:1 rgba(124, 122, 246, 255))
}</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QWidget" name="widget" native="true">
        <property name="minimumSize">
         <size>
          <width>300</width>
          <height>750</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(130, 134, 139);</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_6">
           <item>
            <spacer name="verticalSpacer">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QPushButton" name="show">
           <property name="minimumSize">
            <size>
             <width>300</width>
             <height>100</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">font: 18pt &quot;Calibri&quot;;
color:rgb(255, 255, 255);
background-color:rgb(144, 144, 144);</string>
           </property>
           <property name="text">
            <string>SAVED FILES</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="ocr_1">
           <property name="minimumSize">
            <size>
             <width>300</width>
             <height>100</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">font: 18pt &quot;Calibri&quot;;
color:rgb(255, 255, 255);
background-color:rgb(144, 144, 144);</string>
           </property>
           <property name="text">
            <string>START OCR</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="ocr_2">
           <property name="minimumSize">
            <size>
             <width>300</width>
             <height>100</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">font: 18pt &quot;Calibri&quot;;
color:rgb(255, 255, 255);
background-color:rgb(144, 144, 144);</string>
           </property>
           <property name="text">
            <string>STOP OCR    </string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="rec1">
           <property name="minimumSize">
            <size>
             <width>300</width>
             <height>100</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">font: 18pt &quot;Calibri&quot;;
color:rgb(255, 255, 255);
background-color:rgb(144, 144, 144);</string>
           </property>
           <property name="text">
            <string>START REC</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="rec2">
           <property name="minimumSize">
            <size>
             <width>300</width>
             <height>100</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">font: 18pt &quot;Calibri&quot;;
color:rgb(255, 255, 255);
background-color:rgb(144, 144, 144);</string>
           </property>
           <property name="text">
            <string>STOP REC</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="new1">
           <property name="minimumSize">
            <size>
             <width>125</width>
             <height>100</height>
            </size>
           </property>
           <property name="cursor">
            <cursorShape>ArrowCursor</cursorShape>
           </property>
           <property name="acceptDrops">
            <bool>true</bool>
           </property>
           <property name="styleSheet">
            <string notr="true">font: 18pt &quot;Calibri&quot;;
color:rgb(255, 255, 255);
background-color:rgb(144, 144, 144);</string>
           </property>
           <property name="text">
            <string>START LUMIN</string>
           </property>
           <property name="autoRepeat">
            <bool>false</bool>
           </property>
           <property name="default">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="new2">
           <property name="minimumSize">
            <size>
             <width>300</width>
             <height>100</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">font: 18pt &quot;Calibri&quot;;
color:rgb(255, 255, 255);
background-color:rgb(144, 144, 144);</string>
           </property>
           <property name="text">
            <string>STOP LUMIN</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="register2">
           <property name="minimumSize">
            <size>
             <width>300</width>
             <height>100</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">font: 18pt &quot;Calibri&quot;;
color:rgb(255, 255, 255);
background-color:rgb(144, 144, 144);</string>
           </property>
           <property name="text">
            <string>REGISTER</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="logout">
           <property name="minimumSize">
            <size>
             <width>300</width>
             <height>100</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">font: 18pt &quot;Calibri&quot;;
color:rgb(255, 255, 255);
background-color:rgb(144, 144, 144);</string>
           </property>
           <property name="text">
            <string> LOGOUT</string>
           </property>
           <property name="icon">
            <iconset>
             <normaloff>logout icon.png</normaloff>logout icon.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>30</width>
             <height>30</height>
            </size>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="hello" native="true">
        <property name="styleSheet">
         <string notr="true">QWidget#hello{
background-color:qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(255, 192, 242, 255), stop:1 rgba(170, 170, 255, 255));}</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_5">
           <item>
            <widget class="QLabel" name="name">
             <property name="styleSheet">
              <string notr="true">color:rgb(0, 0, 0);
font: 16pt &quot;Myanmar Text&quot;;</string>
             </property>
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QLabel" name="label">
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
